.github/workflows/chromatic.yml

```diff
@@ -1,18 +1,17 @@
-name: "Chromatic"
-on:
-  push
+name: 'Chromatic'
+on: push
 
 jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - run: yarn
-    - run: yarn gatsby telemetry --disable
-    - run: yarn build
-    - run: yarn build-storybook
-    - uses: chromaui/action@releases/v1
-      with:
-        appCode: 92d6oc1qsrc
-        token: ${{ secrets.GITHUB_TOKEN }}
-        storybookBuildDir: storybook-static
+      - uses: actions/checkout@v1
+      - run: yarn
+      - run: yarn gatsby telemetry --disable
+      - run: yarn build
+      - run: yarn build-storybook
+      - uses: chromaui/action@v1
+        with:
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```

Comment:
This lives in the repo under "Settings" -> "Secrets"

Reply:
Nice!